### ISSUE TITLE
Upload qc back to Hakai database

### ISSUE THREAD

**Issue by JessyBarrette** [emilyhaughton/koeye-stream-timeseries#1]
I started a little notebook to run the conversion between your data packages qc to send back to the database:

https://colab.research.google.com/drive/15VeHU-oV_DUZLY2Dbe8dB0v3FFIg1lUQ?usp=sharing

Steps:

- [ ] Read qced csv from data package
- [ ] timezone = UTC-7:00
- [ ] map variables to original variables
- [ ] Convert data to the original format
- [ ] Convert timestamp to expected time
- [ ] Test upload to db

**Comment by JessyBarrette** [comment]
@emilyhaughton the value presented in your data package does it corresponds to the mean value within the original data?

**Comment by emilyhaughton** [comment]
@JessyBarrette;

--yes, it corresponds to the mean value within the original data
--timezone = PST
--the shift in data is usually not common practice, but mostly happens when a standalone water level sensor has been re-deployed at a different elevation, so a correction needs to be applied to the time-series so it reflects a consistent depth 
          -- it also happens with the realtime PLS water level sensors on Calvert when a sensor fails and another one is installed but in a slightly different depth/location

I don't know that the tool needs to support this function, but it will come up from time to time so having a way to handle it would be good...Could be sussed out by having a rate of change check that auto flags as SVC...Then I could pull out those values, adjust them, and resubmit?

**Comment by JessyBarrette** [comment]
yeah I need to think of something, I could see a way to fix them during the standalone data submissions but for the realtime that wouldn't work. Adding a feature in the qc tool for that could be doable but I don't think this is the purpose of the qc tool.

I'll think about it.

**Comment by JessyBarrette** [comment]
For the timezone PST. You mean a constant `UTC−07:00` or this is local time which includes the daylight savings.

**Comment by emilyhaughton** [comment]
Constant UTC-07:00

**Comment by JessyBarrette** [comment]
Great that saves some headaches!!
